### PR TITLE
Add pending tasks and deploying nodes history

### PR DESCRIPTION
**Pull request by Mykhailenko**

### CODE REVIEW

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java [hunk 1]
```diff
@@ -354,8 +354,8 @@ public void onFailure(Throwable caught) {
      */
     private void fetchStatHistory() {
         String range = "";
-        String[] sources = new String[] { "BusyNodesCount", "FreeNodesCount", "DownNodesCount", "AvailableNodesCount",
-                                          "AverageActivity" };
+        String[] sources = new String[] { "BusyNodesCount", "FreeNodesCount", "DeployingNodesCount", "DownNodesCount",
+                                          "NumberPendingTasks", "AvailableNodesCount", "AverageActivity" };
```
Comment (amouhoub) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java]: Same as in the other PR,
`PendingTasksCount` instead.
All the other metrics are XXXCount as well !

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMStatsView.java [hunk 2]
```diff
@@ -332,7 +341,9 @@ public void statsUpdated(Map<String, StatHistory> values) {
 
         StatHistory freeNodes = values.get("FreeNodesCount");
         StatHistory busyNodes = values.get("BusyNodesCount");
+        StatHistory deployingNodes = values.get("DeployingNodesCount");
         StatHistory downNodes = values.get("DownNodesCount");
+        StatHistory pendingTasks = values.get("NumberPendingTasks");
```
Comment (amouhoub) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMStatsView.java]: Same as above

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMStatsView.java [hunk 3]
```diff
@@ -332,7 +341,9 @@ public void statsUpdated(Map<String, StatHistory> values) {
 
         StatHistory freeNodes = values.get("FreeNodesCount");
         StatHistory busyNodes = values.get("BusyNodesCount");
+        StatHistory deployingNodes = values.get("DeployingNodesCount");
```
Comment (amouhoub) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMStatsView.java]: Maybe match the string values with the variable names freeNodes --> freeNodesCount ?, etc...

Comment (Mykhailenko) [reply rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMStatsView.java]: I am not getting the idea. What do u want me to rename?

**Pull request merged**